meshtastic/tests/test_node_runtime_transport.py

```
@@ -0,0 +1,1287 @@
+    _NodeDeleteChannelRuntime,
+    _NodePositionTimeCommandRuntime,
+)
+from meshtastic.protobuf import admin_pb2, channel_pb2, mesh_pb2, portnums_pb2
+from meshtastic.util import Acknowledgment
+
+
+# Sentinel exception for testing error paths (avoid bare Exception in tests)
+class _SentinelError(Exception):
+    """Sentinel exception for testing _raise_interface_error behavior."""
+
+    pass  # pylint: disable=unnecessary-pass
```

Ruff (0.15.6)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)